src/components/Map/Map.tsx

```diff
@@ -447,7 +447,7 @@ export default function Map(props: MapProps): JSX.Element {
           mapStyle={MapViewStyles[mapViewStyle]}
           initialViewState={{
             bounds: hasEntities ? [options.bbox.lowerLeft, options.bbox.upperRight] : undefined,
-            fitBoundsOptions: hasEntities ? { padding: 20, linear: true } : undefined,
+            fitBoundsOptions: hasEntities ? { padding: 20 } : undefined,
```

Comment:
Why this change? Not sure what the behavioral change is.

Reply:
I think that linear prop was not doing anything because it's not a valid option to pass, but will re-check